### ISSUE TITLE
Pre-built amalgamation for android 64bit

### ISSUE THREAD

**Issue by I3orn2FLY** [apache/incubator-mxnet#11280]
Is there anyway to get already pre-build libmxnet_predict.so of mxnet v1.0.0 or higher of for Android armv8/aarch64? I had too much troubles building amalgamation by myself.

**Comment by ThomasDelteil** [comment]
Please use the forum https://discuss.mxnet.io to ask question and limit github issues to bug report, thanks!

We test the builds in the CI, including amalgation, running in docker containers. One thing I can suggest is to simply run the amalgation build in the same docker image locally.

Have a look here: https://github.com/apache/incubator-mxnet/tree/master/ci

```
ci/build.py --docker-registry mxnetci --platform ubuntu_cpu --shm-size 500m /work/runtime_functions.sh build_ubuntu_amalgamation_min
```

**Comment by kalyc** [comment]
@I3orn2FLY thanks for submitting this issue
@sandeep-krishnamurthy could you add label "Question", "Android" to this?

**Comment by ThomasDelteil** [comment]
@KellenSunderland @lebeg @marcoabreu @KellenSunderland any luck for an Android amalgation build?

**Comment by safrooze** [comment]
@sandeep-krishnamurthy I don't think this qualifies as a question. The user is simply asking the MXNet team to release the amalgamation binary for android as part of the normal release cycle. I recommend to remove Question label and add Feature Request.

**Comment by KellenSunderland** [comment]
I would also see this as more of a feature request.  Apache releases
actually don't generally contain binaries.  These are sometimes provided to
users for convenience.

On the topic of amalgamation I'm not sure what the support status is for
it.  We've recommended a few times that we should deprecate the feature.

On Aug 1, 2018 4:17 AM, "Sina Afrooze" <notifications@github.com> wrote:

@sandeep-krishnamurthy <https://github.com/sandeep-krishnamurthy> I don't
think this qualifies as a question. The user is simply asking the MXNet
team to release the amalgamation binary for android as part of the normal
release cycle. I recommend to remove Question label and add Feature Request.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/apache/incubator-mxnet/issues/11280#issuecomment-409426815>,
or mute the thread
